esi_leap/api/controllers/v1/lease.py

```diff
@@ -155,26 +155,27 @@ def _lease_get_all_authorize_filters(cdict,
 
         if view == 'all':
             policy.authorize('esi_leap:lease:lease_admin', cdict, cdict)
-            possible_filters['owner'] = owner
+            possible_filters['owner_id'] = owner_id
             possible_filters['project_id'] = project_id
         else:
             policy.authorize('esi_leap:lease:get', cdict, cdict)
 
-            if owner:
-                if cdict['project_id'] != owner:
+            if owner_id:
+                if cdict['project_id'] != owner_id:
                     policy.authorize('esi_leap:lease:lease_admin',
                                      cdict, cdict)
 
-                possible_filters['owner'] = owner
+                possible_filters['owner_id'] = owner_id
                 possible_filters['project_id'] = project_id
             else:
                 if project_id is None:
                     project_id = cdict['project_id']
-                elif project_id != cdict['project_id']:
-                    policy.authorize('esi_leap:lease:lease_admin',
-                                     cdict, cdict)
-
-                possible_filters['project_id'] = project_id
+                    possible_filters['project_or_owner_id'] = project_id
```

Comment:
is github being weird or is this tabbed in when it shouldnt be

Reply:
Ah, the indent is correct - we only set 'project_or_owner_id' if the user has decided *not* to filter by project_id. Thanks for the review!

Reply:
oh i was being dumb and read the old line with the new else statement and was confused. All else considered yes it looks fine.